designs/login/index.md

```diff
@@ -0,0 +1,38 @@
+---
+has_children: true
+nav_order: 2
+parent: Home
+layout: default
+title: Login
+has_toc: false
+---
+
+# Login Screen
+
+Unauthenticated users will be redirected to the login screen.
+
+Direct authentication with Kerberos token should be available. If a user has a valid token, the authentication screen will
+show the username and realm which will be used to authenticate against Beaker.
+The user can click 'Log In' to perform the authentication using the token.
+
+![](img/login_kerberos.png)
+
+For those who do not have any form of automatic authentication configured, or for those who click the 'Use another account'
```

Comment:
I don't think this first comma is needed.

Reply:
thanks, fixed